### PR TITLE
Add SQL statement to find current chunk interval

### LINKED ISSUE #406: [Update] SQL statement to find current chunk interval
Eventually, an information view should specify the current chunk interval.  Until then, we should let folks know how to do this in the docs:

> SELECT h.table_name, c.interval_length   FROM _timescaledb_catalog.dimension c JOIN _timescaledb_catalog.hypertable h ON h.id = c.hypertable_id;

Should likely add this to hypertable section on specifying dimension, as well as API reference.

### PR DESCRIPTION
**Pull request by tylerfontaine**
This adds a SQL query against _timescaledb_catalog.hypertable and _timescaledb_catalog.dimension to get the current interval length for hypertables, since this is not shown in the current information views. It is added both to the API reference for timescaledb_information.hypertable and the Hypertable page of Using TimescaleDB.

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -2148,6 +2148,13 @@ WHERE table_schema='public' AND table_name='metrics';
 (1 row)
 ```
 
+If you want to see the current interval length for your hypertables, you can check the `_timescaledb_catalog` as follows. Note that for time-based interval lenghts, these are reported in microseconds.
+
+```Sql
+SELECT h.table_name, c.interval_length   FROM _timescaledb_catalog.dimension c
```
Comment (solugebefola) [inline api.md:2154]: Same as comment on previous PR w.r.t. SQL query line format 😸

File: api.md [hunk 2]
```diff
@@ -2148,6 +2148,13 @@ WHERE table_schema='public' AND table_name='metrics';
 (1 row)
 ```
 
+If you want to see the current interval length for your hypertables, you can check the `_timescaledb_catalog` as follows. Note that for time-based interval lenghts, these are reported in microseconds.
+
+```Sql
```
Comment (solugebefola) [inline api.md:2153]: I _think_ these language designations might need to be all lowercase "sql"

File: api.md [hunk 3]
```diff
@@ -2148,6 +2148,13 @@ WHERE table_schema='public' AND table_name='metrics';
 (1 row)
 ```
 
+If you want to see the current interval length for your hypertables, you can check the `_timescaledb_catalog` as follows. Note that for time-based interval lenghts, these are reported in microseconds.
```
Comment (solugebefola) [inline api.md:2151]: Can you split this line up?
Also
```suggestion
If you want to see the current interval length for your hypertables, you can check the `_timescaledb_catalog` as follows. Note that for time-based interval lengths, these are reported in microseconds.
```

File: using-timescaledb/hypertables.md [hunk 4]
```diff
@@ -108,6 +108,13 @@ taken if you make heavy use of expensive index types (e.g., some
 PostGIS geospatial indexes).  During testing, you might check your
 total chunk sizes via the [`chunk_relation_size`][chunk_relation_size] function.
 
+If you want to see the current interval length for your hypertables, you can check the `_timescaledb_catalog` as follows. Note that for time-based interval lenghts, these are reported in microseconds.
```
Comment (solugebefola) [inline using-timescaledb/hypertables.md:111]:
```suggestion
If you want to see the current interval length for your hypertables, you can check the
 `_timescaledb_catalog` as follows. Note that for time-based interval lengths, these are
  reported in microseconds.
```

File: api.md (hunk 1 above)
Comment (tylerfontaine) [reply api.md:2154]: I thought I put this in line with the example from lines 2142 and 2143 - but happy to reformat in a different way. Should the ON be on a new line?

File: api.md (hunk 2 above)
Comment (tylerfontaine) [reply api.md:2153]: You're right - fixing these.

File: api.md (hunk 3 above)
Comment (tylerfontaine) [reply api.md:2151]: Yep, will fix.

File: using-timescaledb/hypertables.md (hunk 4 above)
Comment (tylerfontaine) [reply using-timescaledb/hypertables.md:111]: Will fix

**Comment by mfreed** [comment]
Just checkin' in, I think some of the above changes still need to be done?

**Pull request closed without merging**